sim/hunter/hunter.go

```diff
@@ -243,6 +244,7 @@ func NewHunter(character *core.Character, options *proto.Player) *Hunter {
 	hunter.AddStatDependency(stats.Agility, stats.MeleeCrit, core.CritPerAgiAtLevel[character.Class][int(character.Level)]*core.CritRatingPerCritChance)
 	hunter.AddStatDependency(stats.Intellect, stats.SpellCrit, core.CritPerIntAtLevel[character.Class][int(character.Level)]*core.SpellCritRatingPerCritChance)
 
+	vanilla.ConstructEmeralDragonWhelpPets(&hunter.Character)
```

Comment:
Can we maybe check if the item is equipped before constructing a pet that will never be used?